SetupCrossTenantRelationshipForTargetTenant.ps1

```diff
@@ -691,34 +723,45 @@ function Run-ExchangeSetupForTargetTenant([string]$targetTenant, [string]$resour
         $global:MigrationEndpoint = New-MigrationEndpoint `
                                         -Name $orgRelName `
                                         -RemoteTenant $resourceTenantDomain `
-                                        -RemoteServer "outlook.office.us" `
+                                        -RemoteServer "outlook.office365.us" `
                                         -ApplicationId $appId `
                                         -AppSecretKeyVaultUrl $appSecretKeyVaultUrl `
                                         -ExchangeRemoteMove:$true
-    }
-    elseif ($Government -eq $true)
-    {
+    } elseif ($Government -eq $true) {
         $global:MigrationEndpoint = New-MigrationEndpoint `
                                         -Name $orgRelName `
                                         -RemoteTenant $resourceTenantDomain `
-                                        -RemoteServer "outlook.office.us" `
+                                        -RemoteServer "outlook.office365.us" `
                                         -ApplicationId $appId `
                                         -AppSecretKeyVaultUrl $appSecretKeyVaultUrl `
                                         -MaxConcurrentMigrations $MigrationEndpointMaxConcurrentMigrations `
                                         -ExchangeRemoteMove:$true
-    }
-    elseif (-not $MigrationEndpointMaxConcurrentMigrations)
-    {
+    } elseif (-not $MigrationEndpointMaxConcurrentMigrations -and $DoD -eq $true) {
+        $global:MigrationEndpoint = New-MigrationEndpoint `
+                                        -Name $orgRelName `
+                                        -RemoteTenant $resourceTenantDomain `
+                                        -RemoteServer "dod-outlook.office365.us" `
+                                        -ApplicationId $appId `
+                                        -AppSecretKeyVaultUrl $appSecretKeyVaultUrl `
+                                        -ExchangeRemoteMove:$true
+    } elseif ($DoD -eq $true) {
+        $global:MigrationEndpoint = New-MigrationEndpoint `
+                                        -Name $orgRelName `
+                                        -RemoteTenant $resourceTenantDomain `
+                                        -RemoteServer "dod-outlook.office365.us" `
+                                        -ApplicationId $appId `
+                                        -AppSecretKeyVaultUrl $appSecretKeyVaultUrl `
+                                        -MaxConcurrentMigrations $MigrationEndpointMaxConcurrentMigrations `
+                                        -ExchangeRemoteMove:$true
+    } elseif (-not $MigrationEndpointMaxConcurrentMigrations) {

```

Comment:
nit: use a single new-migrationendpoint cmdlet, and supply params as a dictionary

Reply:
Will look into this in the future.